virtual-desktop/src/app/authentication-manager/authentication-manager.service.ts

```diff
@@ -178,6 +183,20 @@ export class AuthenticationManager {
       }).catch((err:any)=> {
         observer.error(err);
       });
+
+      this.HTTP.get<any>(ZoweZLUX.uriBroker.pluginConfigUri(ZoweZLUX.pluginManager.getDesktopPlugin(),'pluginData/app', undefined)).subscribe(res => {
```

Comment:
rather than `undefined`, if you give it a specific file name, you can get back an array like I have in the below comment.

Reply:
Using Option 1